BUILDING.md

```diff
@@ -163,7 +163,8 @@ $ [sudo] make install
 
 Prerequisites:
 
-* [Python 2.6 or 2.7](https://www.python.org/downloads/)
+* [Python 2.6 or 2.7](https://www.python.org/downloads/)  
+  *Note*: It is recommanded to include the "Add python.exe to path" option
```

Comment:
recommanded -> recommended ?

Is it a too soft word? Does not build fail without this option?